watchers/factory.go

```diff
@@ -146,5 +147,6 @@ func newChaosPodCacheOptions(disruption *v1beta1.Disruption) (k8scache.Options,
 		SelectorsByObject: k8scache.SelectorsByObject{
 			&corev1.Pod{}: {Label: labels.SelectorFromValidatedSet(ls)},
 		},
+		Namespace: disruption.Namespace,
```

Comment:
This should be the chaos namespace because this means to watch the chaos pods

Reply:
🙈